linux_os/guide/system/logging/ensure_rsyslog_log_file_configuration/rsyslog_files_groupownership/rule.yml

```diff
@@ -50,3 +50,15 @@ ocil: |-
     <tt>/etc/rsyslog.conf</tt> and typically all appear in <tt>/var/log</tt>.
     To see the group-owner of a given log file, run the following command:
     <pre>$ ls -l <i>LOGFILE</i></pre>
+
+template:
+  name: rsyslog_logfiles_attributes_modify
+  vars:
+    attribute: groupowner
+    parameter: file_groupowner_logfiles_value
```

Comment:
The idea here started with group/owner name in parameter, and gid/uid in value, my assumption was that if we keep group/owner names, will be more obvious/readable the idea of the rules/checks/remediations, and if, god forbid, something strange happens in future with the numerical ids that duplication will come in handy to reveal the problem and keep the idea.
But I agree that this might be considered over-engineering and if you insist will drop parameter for value

Reply:
Thanks for the clarification. I would be fine either with names or ids, but we should choose one and use it everywhere instead of both names and ids.